app/src/organisms/RobotSettingsCalibration/CalibrationDetails/OverflowMenu.tsx

```diff
@@ -137,15 +148,19 @@ export function OverflowMenu({
   ): void => {
     e.preventDefault()
     if (!isRunning) {
-      if (calType === 'pipetteOffset') {
-        if (applicablePipetteOffsetCal != null) {
-          // recalibrate pipette offset
-          startPipetteOffsetCalibration({
-            withIntent: INTENT_RECALIBRATE_PIPETTE_OFFSET,
-          })
+      if (calType === 'pipetteOffset' && pipetteName != null) {
+        if (isGen3Pipette) {
```

Comment:
Unexpected any value in conditional. An explicit comparison or type cast is required.

```suggestion
        if (Boolean(isGen3Pipette)) {
```

Reply:
I didn't realize this is required but makes sense to ensure that `isGen3Pipette` is a boolean